.github/workflows/docs.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
           node-version: 10
       - uses: actions/setup-python@v2
         with:
-          python-version: 3.6
+          python-version: 3.8
```

Comment:
3.6 is no longer available on GitHub actions. 3.8 isn't the latest either but there are breaking changes which would mean we have to uptake other changes that I wanted to do right now.